docs/ensnode.io/src/components/organisms/ExamplesPage.tsx

```diff
@@ -0,0 +1,309 @@
+                <svg
+                  className="mx-auto h-12 w-12 text-gray-400"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  stroke="currentColor"
+                >
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    strokeWidth="2"
+                    d="M9.172 16.172a4 4 0 015.656 0M9 12h6m-6-4h6m2 5.291A7.962 7.962 0 0112 15c-2.34 0-4.5-.9-6.172-2.172M15 21H3a2 2 0 01-2-2V5a2 2 0 012-2h12a2 2 0 012 2v16a2 2 0 01-2 2z"
+                  />
+                </svg>
+                <h3 className="mt-2 text-sm font-medium text-gray-900">No queries found</h3>
+                <p className="mt-1 text-sm text-gray-500">
+                  Try adjusting your search or filter criteria.
+                </p>
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+interface QueryCardProps {
+  example: SavedQuery;
+}
+
+const QueryCard: React.FC<QueryCardProps> = ({ example }) => {
+  const slugify = (text: string): string => {
+    return text
+      .toLowerCase()
+      .replace(/[^a-z0-9]+/g, "-")
+      .replace(/(^-|-$)/g, "");
+  };
+
+  const handleInspect = () => {
+    const slug = slugify(example.name);
+    window.location.href = `/examples/${slug}`;
+  };
+
+  const createGraphiQLUrl = (query: string, variables: string): string => {
+    try {
+      const baseUrl = "https://admin.ensnode.io/gql/subgraph-compat";
+      const params = new URLSearchParams({
+        query: query.trim(),
+        variables: variables.trim(),
+      });
+      return `${baseUrl}?${params.toString()}`;
+    } catch (err) {
+      console.error("Failed to create GraphiQL URL: ", err);
+      return "https://admin.ensnode.io/gql/subgraph-compat";
+    }
+  };
+
+  return (
+    <div className="bg-white rounded-lg shadow-md border border-gray-200 hover:shadow-lg transition-all duration-200 hover:-translate-y-1 flex flex-col p-6">
+      <div className="mb-3">
+        <span className="inline-flex items-center px-2.5 py-0.5 rounded-full text-xs font-medium bg-blue-100 text-[var(--color-primary)]">
+          {example.category}
+        </span>
+      </div>
+
+      <h3 className="text-lg font-semibold text-gray-900 mb-2">{example.name}</h3>
+
+      <p className="text-gray-600 text-sm mb-6 overflow-ellipsis line-clamp-3">
+        {example.description}
+      </p>
+
+      <div className="flex gap-2 mt-auto">
+        <a
+          href={createGraphiQLUrl(example.query, example.variables)}
+          target="_blank"
+          rel="noopener noreferrer"
+          className="inline-flex flex-1 justify-center items-center px-3 py-2 border border-transparent text-sm font-medium rounded-md text-white bg-[var(--color-primary)] hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 transition-colors"
+        >
+          <svg
```

Comment:
Instead of using an arrow SVG, it would look much better with the ENSAdmin icon.